extensions/custom_templates/custom_templates.syntax-extension

```diff
@@ -8,19 +8,26 @@ contexts: !merge
     - include: literal-string-template-custom-tags
     - include: styled-components
 
+  expression-begin: !prepend
+    # Keep the slow backreferences out of this frequently-used context.
+    - match: (?=`)
+      set:
+        - include: literal-string-template-custom-comments
+        - include: literal-string-template
+
   literal-string-template-custom-tags: !foreach
     in: !argument [tags, {}]
     as: [tag, include]
     value:
-      match: !format '{tag}(?=\s*`)'
+      match: !format '(?:{tag})(?=\s*`)'
       scope: variable.function.tagged-template.js
       set: !include_resource Packages/JSCustom/extensions/custom_templates/template.yaml
 
   literal-string-template-custom-comments: !foreach
     in: !argument [comments, {}]
     as: [tag, include]
     value:
-      match: !format '/\*\s*{tag}\s*\*/(?=\s*`)'
+      match: !format '(?<=/\*(?:{tag})\*/)'
```

Comment:
Could this possibly be `(?<=/\*\s?(?:{tag})\s?\*/)` instead? That would allow for one space inside the comment and the performance should still be okay (unlike with `\s*`).

Reply:
Unfortunately, Oniguruma only supports fixed-width lookbehinds. The performance hit is due to using Oniguruma at all instead of Sublime's custom regexp engine.

Reply:
Ah that is unfortunate. Will Oniguruma accept this regexp?

```
(?:(?<=/\*(?:{tag})\*/)|(?<=/\* (?:{tag}) \*/))
```

I just strongly prefer to have spaces in inline comments to make them more readable.

Reply:
Seems reasonable: #77.

Reply:
Cool, thanks for taking my suggestion :)